src/booking/BookingCalendar.tsx

```diff
@@ -51,7 +52,18 @@ export const BookingCalendar: FunctionComponent<BookingCalendarProps> = ({
       {Array.from(data.entries()).map(([month, dateLocationsPairsForMonth]) => (
         <CalendarSectionContainer key={month}>
           <div className="MonthSection">
-            <h2>{t("calendar.month", { monthString: month })}</h2>
+            <h2>
+              {
+                parse(
+                  "1 " + month,
+                  "d MMMM yyyy",
```

Comment:
nit: remove the day from the parsing format (ie: change to `MMMM yyyy`) so that we can remove the "1 " + string concatenation from line 58.